### DISCUSSION TITLE
Does Github education or Github classroom have an API?

### DISCUSSION THREAD

**Discussion by ytbryan** [community/community#28542] [GitHub Education]
Dear moderator,

Does Github education or Github classroom have an API? 

Maybe a CLI that allows the generation of a classroom or an assignment etc. 

Bryan

**Comment by samus-aran** [comment]
Hello @ytbryan ! 👋  At this moment we don't have Classroom APIs for classrooms/assignments and a CLI is a very good suggestion so thank you for requesting this. ✍️ 

There is [another discussion](https://github.com/community/community/discussions/18978) that potentially describes what you're looking for created by the community, perhaps check those out to assist with you Classroom workflow.
